src/WorkerIndexing/WorkerIndexingHelper.cs

```diff
@@ -0,0 +1,50 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.WorkerIndexing
+{
+    internal class WorkerIndexingHelper
+    {
+        internal static IEnumerable<RpcFunctionMetadata> IndexFunctions(string baseDir)
+        {
+            List<RpcFunctionMetadata> indexedFunctions = new List<RpcFunctionMetadata>();
+
+            InitialSessionState initial = InitialSessionState.CreateDefault();
+            Runspace runspace = RunspaceFactory.CreateRunspace(initial);
+            runspace.Open();
+            System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create();
+            ps.Runspace = runspace;
+
+            ps.AddCommand("Get-FunctionsMetadata").AddArgument(baseDir);
```

Comment:
Please make this a constant. Here is an example: https://github.com/Azure/azure-functions-powershell-worker/blob/ac95680f3d905a85352410d942f66e536d9ddde5/src/DependencyManagement/PowerShellGalleryModuleProvider.cs#L82

Reply:
Done